microsoft-edge/toc.yml

```diff
@@ -373,22 +373,44 @@
 # equivalent card: Progressive Web Apps =======================================
     - name: Progressive Web Apps
       items:
-        - name: Progressive Web Apps overview
+        - name: Overview of Progressive Web Apps (PWAs)
           href: progressive-web-apps-chromium/index.md
+        - name: The user experience of PWAs
+          href: progressive-web-apps-chromium/ux.md
+        - name: How to
```

Comment:
I'm no fan of these generic buckets.  I'd look for a feature-based outline.  Users are looking for feature areas, not doc-department info-types.

Reply:
Now that the PR is Ready for Review, I'm looking at this again, this use of the How-to bucket to wrap 90% of the articles and keep the Intro level clean.